resources/qml/Toolbar.qml

```diff
@@ -135,4 +156,15 @@ Item {
 
         visible: toolHint.text != "";
     }
+
+    UM.SettingPropertyProvider
+    {
+        id: machineExtruderCount
+
+        containerStackId: Cura.MachineManager.activeMachineId
+        key: "machine_extruder_count"
+        watchedProperties: [ "value" ]
+    }
+
+    UM.I18nCatalog { id: catalog; name: "cura" }
```

Comment:
(Not necessary) should we use the same coding style as above? (expand to multiple lines)